### DISCUSSION TITLE
What about add more langluages?

### DISCUSSION THREAD

**Discussion by deverydo** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#520] [Ideas]
Can you add more langluages?

**Comment by im-coder-lg** [comment]
Oh, sorry for the late reply, I noticed but I forgot. Yes, you _can_ add more languages **only** if the Google Assistant SDK  and the client supports it.

**Comment by Melvin-Abraham** [comment]
Although the official version does have more language choices, Google Assistant SDK has limited support for languages. You can still talk to the Assistant in any language you want and it will continue the conversation without any hassle. But based on your concern I assume you are referring to the application's display language. There are plans to work on multiple languages for the same after the next release...
